src/hypertable.c

```diff
@@ -1007,6 +1012,16 @@ hypertable_create(PG_FUNCTION_ARGS)
 		tablespace_attach_internal(&tspc_name, table_relid, false);
 	}
 
+	/* Migrate data from the main table to chunks */
+	if (table_has_data)
+	{
+		ereport(NOTICE,
+				(errmsg("migrating data to chunks..."),
```

Comment:
I would change errmsg to be `migrating data to chunks... this may take a while`

Reply:
I put that in the hint. Are you suggesting I move that to the message?

Reply:
Yes I think that's better